gmusicapi/clients/musicmanager.py

```diff
@@ -385,7 +385,8 @@ def download_song(self, song_id):
 
     @utils.accept_singleton(basestring)
     @utils.empty_arg_shortcircuit(return_code='{}')
-    def upload(self, filepaths, transcode_quality='320k', enable_matching=False):
```

Comment:
we may want to keep the existing order of the params to avoid an unnecessary breaking change for those calling positionally.